CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - More errors now set the exit code.
 - Added support for error({any}) to the Roblox standard library.
 
+### Fixed
+- Fixed several required parameters.

```

Comment:
This isn't descriptive enough. Please read similar changelog entries to see what is expected.